examples/webpack-example/package.json

```diff
@@ -8,18 +8,16 @@
   },
   "scripts": {
     "start": "webpack-dev-server --config webpack-dev-server.config.js --progress --inline --colors",
-    "build": "webpack --config webpack-production.config.js --progress --colors"
+    "build": "webpack --config webpack-production.config.js --progress --colors",
+    "lint": "eslint --ext .js src && echo \"eslint: no lint errors\""
```

Comment:
Don't need the `--ext .js` unless we're including other types too (such as `jsx`)